app/common/getDataFactory.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+angular.module('batApp')
+.factory('getDataFactory', function(configuration, $resource) {
+    return {
+        sendData: function(url) {
```

Comment:
create specific api function. for each api on specific function. dont make a generic function for all api